### ISSUE TITLE
Discuss if renaming nested_rbac.bicep to nested_roleAssignments.bicep

### ISSUE THREAD

**Issue by eriqua** [Azure/ResourceModules#1062]
## Description

Discuss if renaming nested_rbac.bicep to nested_roleAssignments.bicep to align to the resource type name.

**Comment by MariusStorhaug** [comment]
Agree, its in line with what it does. Also thinking in that case that `nested_` should be `extension_`
